### PR TITLE
Atomic Store: Restore Theme Selection

### PR DESCRIPTION
**Pull request by donpark**
This PR restores theme selection screen to Atomic Store signup flow.

Only themes marked with `store` in its `design` field will be displayed.

## Test Plan

1. Start Atomic Store signup flow (`/start/atomic-store`) and select Store option.
2. You should see theme selection like image below.

screenshot_368

### CODE REVIEW

**Comment by donpark** [comment]
Question: Why isn't *Storefront* theme among suggested choices?

**Comment by kellychoffman** [comment]
> Question: Why isn't Storefront theme among suggested choices?

Good question! It is because Storefront is not in the WP.com theme repo, so it is only available once the Atomic transfer process is completed.

**Comment by tiagonoronha** [comment]
> It is because Storefront is not in the WP.com theme repo

Since they are signing up to get a store from the get go, can't we offer Storefront and child themes here?

**Comment by justinshreve** [comment]
> Since they are signing up to get a store from the get go, can't we offer Storefront and child themes here?

Agreed - at the very least it feels like we should hack in a Storefront image/option here and have it be the default selection. It'll be weird to get through the sign-up flow, go to the customizer step, and then be told that Storefront is better when they were not offered it before.

**Comment by kellychoffman** [comment]
If we can do that, let's go for it. I thought there was a technical reason why it couldn't be done.

**Comment by kellychoffman** [comment]
However, they won't see the "Storefront is better" messaging if they have a theme that is WC compatible, which these all are.

**Comment by donpark** [comment]
Thanks for the answer which seems to be "Let's do it!" and PR approve.
Let's handle the Nike thing in another issue to keep things tidy. :-)

File: client/signup/steps/theme-selection/index.jsx [hunk 1]
```diff
@@ -91,6 +91,10 @@ class ThemeSelectionStep extends Component {
 	}
 
 	shouldSkipStep() {
```
Comment (lamosty) [inline client/signup/steps/theme-selection/index.jsx]: We can just remove this function completely now. :)

File: client/signup/steps/theme-selection/index.jsx [hunk 2]
```diff
@@ -124,15 +128,22 @@ class ThemeSelectionStep extends Component {
 			return null;
 		}
 
+		const storeSignup = this.isStoreSignup();
```
Comment (lamosty) [inline client/signup/steps/theme-selection/index.jsx]: I think we prefer to prefix variables/functions names with `is` if it holds a boolean value.

**Pull request merged**